crates/compiler/parse/src/parser.rs

```diff
@@ -1376,6 +1375,64 @@ macro_rules! and {
     };
 }
 
+/// Similar to `and`, but we modify the min_indent of the second parser to be
+/// 1 greater than the line_indent() at the start of the first parser.
+#[macro_export]
+macro_rules! indented_seq {
+    ($p1:expr, $p2:expr) => {
+        move |arena: &'a bumpalo::Bump, state: $crate::state::State<'a>, _min_indent: u32| {
+            let start_indent = state.line_indent();
+
+            // TODO: we should account for min_indent here, but this doesn't currently work
```

Comment:
Is there an issue to track occurrences of this, so we know when all are resolved?

Reply:
This is very closely related to #3655. My intent is to fully implement my "#3" suggestion there, which will eliminate uses of `increment_min_indent`.